pyproject.toml

```diff
@@ -32,9 +32,10 @@ pythonpath = [
     "tests",
 ]
 markers = [
-    "unit",  # offline
-    "integration",  # hardhat fork
-    "integration_holesky" # holesky fork
+    "unit", # offline
+    "integration", # hardhat fork
+    "integration_holesky", # holesky fork
+    "integration_manual" # manual run only
```

Comment:
Hm. Why it is manual only?
Maybe create bash script that can automatize everything?

Reply:
Made it connect to the gnosis testnet, however I need to see messages there continuously